components/comment.vue

```diff
@@ -21,7 +21,7 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from "nuxt-property-decorator"
+import { Component, Prop, Vue } from "vue-property-decorator"
```

Comment:
@kevinmarrec let's here stick to nuxt-property-decorator for now, as it will be good to see what additional value it can bring to the table. Vue property decorator is only concerned about vue decorators, the idea with nuxt property decorator is to bring vue, vuex and possibly  vue-router decorators together in nuxt style.

Reply:
@husayt Well, I suggest stick to `vue-property-decorator` causes `nuxt-property-decorator` makes Vue methods like `mounted`, `created`, etc... abstracted to the class level without any autocompletion or typechecking.

That's why I migrated these method definitions into the `@Component` options.

People can use custom libraries for `vuex` things externally.

IMO, using `nuxt-property-decorator` is not worth it but I let you argument on this